ui/README.md

```diff
@@ -1,4 +1,14 @@
-# Vue 3 + Vite
+# Armadillo UI
+
+## Test as a UI developer
+
+- `cd ui/`
+- make sure you have `yarn` installed.
+- `yarn test --watch`
+- change one of `src/**`
+- change corresponding `tests/unit/**`
```

Comment:
I know what you mean by this, but it can be preserved as confusing at first... Perhaps it's better to state the last points as "do your developer things but also remember to write and/or update the tests" (perhaps different wording)

Reply:
@SietsmaRJ please check the rewrite